### PR TITLE
[ADD] Colors to calendar task categories

### PR DESCRIPTION
**Pull request by eilst**

### CODE REVIEW

File: project_event/views/project_task_view.xml [hunk 1]
```diff
@@ -320,4 +321,28 @@
               parent="project_event_menu"
               action="action_project_event_task"
               sequence="30"/>
+    
+    <record id="calendar_color_project_task_color_view" model="ir.ui.view">
```
Comment (rimbendhaou) [inline project_event/views/project_task_view.xml]: @eilst Why you didn't put this code in the same view : project_event_task_calendar

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/models/project_task.py [hunk 2]
```diff
@@ -29,6 +29,7 @@ class Task(models.Model):
     task_category_id = fields.Many2one(
         'task.category',
         string='Category',
+        default=lambda self: self.env.ref('project_event.task_category_uncategorized').id,
```
Comment (rimbendhaou) [inline project_event/models/project_task.py]: where is this external id: project_event.task_category_uncategorized?

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/views/project_task_view.xml [hunk 3]
```diff
@@ -267,12 +268,24 @@
...
+	   https://github.com/odoo/odoo/pull/26750
+    -->
     <record model="ir.ui.view" id="project_event_task_calendar">
         <field name="name">project.event.task.calendar</field>
         <field name="model">project.task</field>
         <field name="arch" type="xml">
             <calendar string="Calendar View" date_start="date_start"
-                      date_stop="date_end">
+                      date_stop="date_end"
+                       js_class = "calendar_colour"
```
Comment (rimbendhaou) [inline project_event/views/project_task_view.xml]: @eilst indentation, no space before and after  =

File: project_event/models/project_task.py [hunk 4]
```diff
@@ -29,6 +29,7 @@ class Task(models.Model):
     task_category_id = fields.Many2one(
         'task.category',
         string='Category',
+        default=lambda self: self.env.ref('project_event.default_category').id,
```
Comment (rimbendhaou) [inline project_event/models/project_task.py]: self.env.ref('task_category_uncategorized').id,

File: project_event/views/project_task_view.xml [hunk 5]
```diff
@@ -319,5 +332,6 @@
               name="Task"
               parent="project_event_menu"
               action="action_project_event_task"
-              sequence="30"/>
+              sequence="30"/>
```
Comment (rimbendhaou) [inline project_event/views/project_task_view.xml]: why this empty spaces?

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/models/project_task.py (hunk 4 above)
Comment (eilst) [reply project_event/models/project_task.py]: done

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**